brayns/common/scene/Scene.cpp

```diff
@@ -517,9 +519,46 @@ VolumeHandlerPtr Scene::getVolumeHandler()
     {
         const std::string& volumeFile =
             _parametersManager.getVolumeParameters().getFilename();
-        if( !_volumeHandler && !volumeFile.empty() )
-            _volumeHandler.reset( new VolumeHandler(
-                _parametersManager.getVolumeParameters(),  volumeFile ));
+        const std::string& volumeFolder =
+            _parametersManager.getVolumeParameters().getFolder();
+
+        if( !_volumeHandler && ( !volumeFile.empty() || !volumeFolder.empty() ))
+        {
+            _volumeHandler.reset(
+                new VolumeHandler( _parametersManager.getVolumeParameters(), TM_MODULO ));
```

Comment:
It took me a while to realize what this TM_MODULO was. I suggest you use enums with a class name, describing the type. See https://github.com/BlueBrain/Fivox/blob/master/fivox/types.h#L80 for examples.